io/shared/src/main/scala/org/scalajs/io/package.scala

```diff
@@ -10,14 +10,14 @@
  * additional information regarding copyright ownership.
  */
 
-package org.scalajs.linker
+package org.scalajs
 
 import scala.concurrent._
 
 import scala.util.{Try, Success, Failure}
 
-package object irio {
-  private[irio] implicit class FutureOps[T](val __self: Future[T]) extends AnyVal {
+package object io {
+  private[scalajs] implicit class FutureOps[T](val __self: Future[T]) extends AnyVal {
```

Comment:
Actually, you can probably avoid the problem altogether if you first get rid of the `io` package, since everything would then be in the `scalajs-linker` artifact anyway.